pages/glossary/highest-possible-contrast.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Highest Possible Contrast
+key: highest-possible-contrast
+---
+
+The highest value when computing the [contrast ratio](https://www.w3.org/TR/WCAG21/#dfn-contrast-ratio) between two sets of colors.
```

Comment:
In my opinion this adds an overhead to read a definition which simply refers to a possible value of color-contrast.

Given this is only used in one place, I suggest we change the wording around this usage & lose this definition.

Reply:
We use it in one place now. But this definition will be useful when we write multiple contrast rules (which we are planning).